gulpfile.js

```diff
@@ -87,7 +87,7 @@ var bundleBaseConfig = {
 var bundles = [{
   // Public-facing website
   name: 'site',
-  entry: './h/static/scripts/site',
```

Comment:
This is is now an unnecessary change. Please get into the habit of reviewing your own diffs to spot stuff like this!